content/en/apps/features/muting/_index.md

```diff
@@ -1,63 +0,0 @@
-
-{{< figure src="muted-family-appearance-on-list.png" link="muted-family-appearance-on-list.png" class="col-3 col-lg-6" >}}
-
-For the cases where families are muted, each of the family members in the people card are also styled as muted, (grey icons and “Muted” labels).A user may add a new person to a muted household. If they do, and choose to leave the family muted, the new person will be automatically muted once created. When a family is unmuted, all individuals in the family will be unmuted at the same time. 
-
-Unmuting an individual person in a muted family works essentially the same way, the action unmutes the entire family because unmuted individuals  are not allowed underneath a muted household.
-
-### Configuration
-
-Through [configuration]({{< relref "apps/reference/app-settings/transitions" >}}), you can specify which form mutes and unmutes, form availability, form questions and options, manager verification and labels used for display and warning messages.
-
-### Muting vs Death Reporting
-
-A project may support both death reporting and muting - they are not mutually exclusive. Death reporting moves the deceased person to a different part of the family members list and does not allow actions. Muting keeps the person in the family members list, allows actions and disables schedules.
-
-|   Death reporting                                                                                        | Muting                                                                                         |
-| -------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
-| <p></p><ul><li>Permanent state</li></ul>                                                                 | <ul><li>Temporary state</li></ul>                                                              |
-| <p></p><ul><li>Only allowed at the individual level</li></ul>                                            | <ul><li>Place, family, or individual level</li></ul>                                           |
-| <p></p><ul><li>Removes schedules</li></ul>                                                               | <ul><li>“Quiets” notifications for schedules</li></ul>                                         |
-| <p></p><ul><li>A deceased individual is removed from the family list</li></ul>                           | <ul><li>A muted individual is not removed from the family list<br></li></ul>                   |
-| <p></p><ul><li><p>No new actions can be performed</p><p>except for one - reverse the death</p></li></ul> | <ul><li>New actions may be performed, but no tasks or notifications will be sent<br></li></ul> |
-| <p></p><ul><li>Manager confirmation configurable</li></ul>                                               | <p></p><ul><li>Manager confirmation configurable</li></ul>                                     |
-
```

Comment:
I would expect migration of muting docs to be done as part of https://github.com/medic/cht-docs/pull/877

Reply:
Yes @eljhkrr , my git pull didn't work but I have removed the muting items.